addons/clickable-quote-authors/addon.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Clickable author links in forum quotes",
+  "description": "Replaces author names in forum quotes with clickable links.",
+  "tags": ["community", "forums"],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["forums"]
+    }
+  ],
+  "settings": [
+    {
+      "name": "Hidden link",
```

Comment:
`curator-link` calls this setting "Style as normal text".

Reply:
Yeah, I wasn't sure what to call it. I've renamed it now.